api/src/main/java/org/apache/iceberg/catalog/Catalog.java

```diff
@@ -28,7 +29,7 @@
 /**
  * Top level Catalog APIs that supports table DDLs and namespace listing.
```

Comment:
Should the comment be "Top-level Catalog API that supports ..." or "Top-level Catalog APIs that support ..."?

Reply:
I think it should be singular API.